src/python/director/drakevisualizer.py

```diff
@@ -289,12 +298,12 @@ def __init__(self, view):
         self.view = view
         self.robots = {}
         self.linkWarnings = set()
-        self.sendStatusMessage('loaded')
         self.enable()
+        self.sendStatusMessage('loaded')
```

Comment:
I switched the order of these lines so that the status message is not sent until *after* the LCM subscriptions are created. Otherwise it's hard to tell when the viewer is ready to start receiving messages.